recipes/cosmic-ray/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cosmic-ray" %}
```

Comment:
@synapticarbors @jakirkham, the package name on pypi is cosmic_ray, but underscores on packages names are ugly :-|, should I still use that?